tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -15,16 +15,16 @@ def __init__(
             self["scrollIntoView"] = scroll_into_view
 
 
-class ViewportOptions(Dict[str, Any]):
+class BoxOptions(Dict[str, Any]):
     def __init__(self, x: float, y: float, width: float, height: float):
-        self["type"] = "viewport"
```

Comment:
Note that this is a backward incompatible change. In which version of Chrome and when exactly will it be released? It would be good to coordinate it together with a Firefox release.

Reply:
We recently added it into our WebDriver BiDi implementation, but we haven't merged it into Chrome yet. Considering Puppeteer uses the WebDriver BiDi implementation directly, we are not generally in a rush to upstream it into Chromium. However, considering this change, we already have plans on doing so immediately.

Reply:
Note that we usually try to avoid backward incompatible changes and I cannot see that we actually agreed on it - maybe it was missed in both the reviews for the BiDi and test changes? So lets hope that not many clients are actually affected by that.

We filed https://bugzilla.mozilla.org/show_bug.cgi?id=1859258 to get our implementation updated, which may still happen for Firefox 120 which will be released November 21.